server/sonar-server/src/test/java/org/sonar/server/computation/ComputationServiceTest.java

```diff
@@ -66,7 +68,7 @@ public void process_project() throws Exception {
     when(steps.orderedSteps()).thenReturn(Arrays.asList(projectStep1, projectStep2, viewStep));
 
     // load report from db and parse it
-    ComputationService sut = new ComputationService(dbClient, steps, activityService);
+    ComputationService sut = new ComputationService(dbClient, steps, activityService, mock(ProjectSettingsFactory.class, Mockito.RETURNS_DEEP_STUBS));
```

Comment:
Is RETURNS_DEEP_STUBS necessary here? The mocked ProjectSettingsFactory should return a true Settings object.